jenkins/test_orchestrator/JenkinsFile

```diff
@@ -0,0 +1,62 @@
+pipeline {
```

Comment:
Since these are workflows, I have structured it under bundle-workflow
Ref: https://github.com/opensearch-project/opensearch-build/pull/423/files#diff-500b943d0ad57b148b713ee7cfbca15515eede9d0e9614d1ef8d9f16f30a0001
`bundle-workflow/jenkins_workflow/integtest/Jenkinsfile`

You could do something similar, `bundle-workflow/jenkins_workflow/test_orchestrator/Jenkinsfile` and also add a Readme.md

Reply:
sure, added in #523